editions/tw5.com/tiddlers/$__StoryList.tid

```diff
@@ -1,5 +0,0 @@
-created: 20220728145919904
-list: Brackets Base64 HelloThere GettingStarted Community
-modified: 20220728145919904
-title: $:/StoryList
-type: text/vnd.tiddlywiki
```

Comment:
This should be a separate PR in order to have a cleaner and easier to follow commit history.

Reply:
So you say, I should add it back and pollute the commit history even more, just to remove it with a separate PR? IMO that doesn't make sense to me

Reply:
Your original PR has been reverted and you are making a change unrelated to this PR. So yes, it does make sense. The story list tiddler needs to be amended to undo what is probably an erroneous commit by Jeremy but does not need to be deleted.